### PR TITLE
fix(dropdown): fix dropdown inside click

### PR DESCRIPTION
**Pull request by ghost**
- fix dropdown inside click
- fixed dropdown's and accordion's tests

Related to: #1933
# PR Checklist

### CODE REVIEW

**Review by Domainv — DISMISSED** [review]

**Comment by OlegDubrovskyi** [comment]
Looks good.
Checked on Firefox, Chrome, IE11, Edge

**Review by valorkin — DISMISSED** [review]

**Comment by ludmilanesvitiy** [comment]
Tested, in FF 63.0 dropdown didn't close by outside click, in IE and Chrome works well. And looks like this PR need additional unit-tests.

**Comment by ghost** [comment]
@ludmilanesvitiy issue in Firefox was fixed, tests was added

File: src/spec/bs-dropdown-container.component.spec.ts [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+    fixture.detectChanges();
+  });
+
+  it('should not be null', () => {
+    expect(component).not.toBeNull();
+  });
+
+  it('should set isOpen property', () => {
+    component.isOpen = true;
+    expect(component.isOpen).toBeTruthy();
```
Comment (Domainv) [inline src/spec/bs-dropdown-container.component.spec.ts]: Seriously?

Comment (valorkin) [reply src/spec/bs-dropdown-container.component.spec.ts]: Cap ;)

**Review by valorkin — APPROVED** [review]

File: src/spec/bs-dropdown-container.component.spec.ts (hunk 1 above)
Comment (ghost) [reply src/spec/bs-dropdown-container.component.spec.ts]: was deleted

**Review by valorkin — APPROVED** [review]

**Pull request merged**